app-modules/analytics/src/Filament/Resources/AnalyticsResourceResource/Pages/CreateAnalyticsResource.php

```diff
@@ -0,0 +1,84 @@
+      in the software, and you may not remove or obscure any functionality in the
+      software that is protected by the license key.
+    - You may not alter, remove, or obscure any licensing, copyright, or other notices
+      of the licensor in the software. Any use of the licensor’s trademarks is subject
+      to applicable law.
+    - Canyon GBS LLC respects the intellectual property rights of others and expects the
+      same in return. Canyon GBS™ and Advising App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    https://www.canyongbs.com or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+*/
+
+namespace AdvisingApp\Analytics\Filament\Resources\AnalyticsResourceResource\Pages;
+
+use Filament\Forms\Form;
+use Filament\Forms\Components\Select;
+use Filament\Forms\Components\Section;
+use Filament\Forms\Components\Checkbox;
+use Filament\Forms\Components\Textarea;
+use Filament\Forms\Components\TextInput;
+use Filament\Resources\Pages\CreateRecord;
+use Filament\Forms\Components\SpatieMediaLibraryFileUpload;
+use AdvisingApp\Analytics\Filament\Resources\AnalyticsResourceResource;
+
+class CreateAnalyticsResource extends CreateRecord
+{
+    protected static string $resource = AnalyticsResourceResource::class;
+
+    public function form(Form $form): Form
+    {
+        return $form
+            ->schema([
+                Section::make()
+                    ->columns()
+                    ->schema([
+                        TextInput::make('name')
+                            ->required()
+                            ->string()
+                            ->unique(ignoreRecord: true),
+                        Textarea::make('description')
+                            ->nullable()
+                            ->string(),
+                        Select::make('category_id')
+                            ->relationship('category', 'name')
+                            ->required(),
+                        Select::make('source_id')
+                            ->relationship('source', 'name'),
+                        TextInput::make('url')
+                            ->nullable()
+                            ->url(),
+                        SpatieMediaLibraryFileUpload::make('thumbnail')
+                            ->nullable(),
+                        Checkbox::make('is_active')
+                            ->label('Active'),
+                        Checkbox::make('is_included_in_data_portal')
+                            ->label('Included in Data Portal'),
```

Comment:
Do `Checkbox` inputs default to false automatically? If not, we should set the default here. Since the columns in th DB do not have a default, we don't want to risk a SQL error on submit.

Reply:
It appears that they are false by default, but I set them to default false in the migration just to make it clear.